.eslintrc.js

```diff
@@ -6,10 +6,12 @@ module.exports = {
   },
   plugins: ['@typescript-eslint/eslint-plugin'],
   extends: [
-    'plugin:@typescript-eslint/eslint-recommended',
+    // Uses the recommended rules from the @typescript-eslint/eslint-plugin
     'plugin:@typescript-eslint/recommended',
-    'prettier',
+    // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
     'prettier/@typescript-eslint',
+    // Must always be last
+    'plugin:prettier/recommended'
```

Comment:
This line is the main improvement here, it activates eslint rules that map to the relevant prettier rules as configured in `.prettierrc` and thanks to that, `eslint --fix` will resolve both eslint AND prettier code formatting issues.